Makefile

```diff
@@ -4,8 +4,6 @@
 # Installation can still be performed with a
 # normal `go install`.
 
-# generated integration test files
-GGEN = ./_generated/generated.go ./_generated/generated_test.go
```

Comment:
What was the failure that made it so this needed to be removed?

Reply:
I assume there will be a lot of them. The code these tests use was ripped out. For example, https://github.com/algorand/msgp/blob/master/_generated/convert_test.go#L13
uses https://github.com/tinylib/msgp/blob/43b487371e0cdd1a3fe1be9a7802d8bce917a1af/msgp/write.go#L118 from the original repository which was removed from our fork in this commit https://github.com/algorand/msgp/commit/546fb5f3a925227e2f999ebc20235f647dcc6e1a